app/views/activities/_status_update.html.slim

```diff
@@ -1,9 +1,10 @@
 li class="activity #{activity.kind}"
-  = image_tag(activity.img_url || '/images/screenshot-missing.png')
+  / = image_tag(activity.img_url || '/images/happyface.png')
+  / = image_tag(activity.img_url || '//avatars.githubusercontent.com/u/11146524')
+  = image_tag(activity.img_url || "#{user.avatar_url}")
```

Comment:
Here you use

```ruby
"#{user.avatar_url}"
```

Because the whole string is just the contents of the variable, it should just be:

```ruby
user.avatar_url
```